_projects/2023/100430897/100430897.Rmd

```diff
@@ -0,0 +1,1048 @@
+    axis.title = element_text(size = 12, color = "black", face = "bold"),
+    panel.grid.minor = element_blank(),
+    panel.grid.major.x = element_blank(),
+    axis.ticks.x = element_line(size = 0.5),
+    axis.text.x = element_text(angle = 0, hjust = 0.5),  # Fondo del gráfico
+    axis.ticks.y = element_line(size = 0.5),
+    axis.text.y = element_text(angle = 0, hjust = 0.5),
+    plot.background = element_rect(fill = "white"),
+    plot.margin = margin(1, 1, 1, 1, "cm"),  # Añade márgenes alrededor del gráfico
+    legend.key.width = unit(1, "cm"),  # Ajusta el ancho de la clave de la leyenda
+    legend.key.height = unit(1, "cm"),  # Ajusta la altura de la clave de la leyenda
+    legend.spacing = unit(0.5, "cm"),  # Ajusta el espacio entre las entradas de la leyenda
+    legend.box.margin = margin(0, 0, 0, 0, "cm")   # Fondo del gráfico
+  ) +
+  guides(
+    color = guide_legend(
+      keywidth = 1, keyheight = 0.5,   # Ajustes de ancho y alto de las muestras de color
+      label.position = "right",  # Coloca las etiquetas de la leyenda en la parte inferior
+      ncol = 4  # Número de columnas en la leyenda
+    )
+  ) +
+  geom_text(aes(x = as.Date("1958-12-01"), y = 90), label = "Eisenhower", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#bf3927") +
+  geom_text(aes(x = as.Date("1961-01-20"), y = 90), label = "Kennedy", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#436983") +
+  geom_text(aes(x = as.Date("1963-11-22"), y = 90), label = "Johnson", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#436983") +
+  geom_text(aes(x = as.Date("1969-01-20"), y = 90), label = "Nixon", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#bf3927") +
+  geom_text(aes(x = as.Date("1974-08-09"), y = 90), label = "Ford", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#bf3927") +
+  geom_text(aes(x = as.Date("1977-01-20"), y = 90), label = "Carter", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#436983") +
+  geom_text(aes(x = as.Date("1981-01-20"), y = 90), label = "Reagan", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#bf3927") +
+  geom_text(aes(x = as.Date("1989-01-20"), y = 90), label = "H.W. Bush", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#bf3927") +
+  geom_text(aes(x = as.Date("1993-01-20"), y = 90), label = "Clinton", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#436983") +
+  geom_text(aes(x = as.Date("2001-01-20"), y = 90), label = "W. Bush", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#bf3927") +
+  geom_text(aes(x = as.Date("2009-01-200"), y = 90), label = "Obama", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#436983") +
+  geom_text(aes(x = as.Date("2017-01-20"), y = 90), label = "Trump", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#bf3927") +
+  geom_text(aes(x = as.Date("2021-01-20"), y = 90), label = "Biden", vjust = -1, hjust = 0.2, size = 3, angle = -90, color = "#436983") 
```

Comment:
In general, try to style your code so that it's shorter. See the HTML output: these lines go out of the page. A maximum width of 80 characters for code is generally advisable.